.circleci/config.yml

```diff
@@ -79,13 +79,16 @@ jobs:
           command: npm install license-checker@25.0.1
       - run:
           name: Check SDK Licenses
-          command: cd ~/src/raiden-ts && license-checker --production --onlyAllow 'MIT;BSD;Apache;MPL;LGPL;ZPL;ISC;WTFPL;Unlicense' --excludePackages 'uuid@2.0.1' --excludePrivatePackages # uuid@2.0.1 has an issue with the license information but it is licensed under MIT
+          working_directory: raiden-ts
```

Comment:
How does this work? Above we define `working_directory: ~/`, so shouldn't this be `src/raiden-ts`?

Reply:
This is outdated. But you are right, actually this should not have worked on this commit. :thinking: